ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -24152,6 +24233,88 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build03
+  cron: 38 23 10 * *
```

Comment:
in https://github.com/openshift/release/blob/771b675615346f57f2190c5736b5abb92f1a56ce/ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml#L12605
periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-azure-upi-p3-f28-destructive
cron: 38 23 1 * *
How about change to 38 23 16 * * ?

Reply:
I changed it properly to distribute the executione time between the archs. We'll get amd64 executing on 1st of each month, the arm64 one on 16th.